src/utils/__tests__/build.test.js

```diff
@@ -25,7 +25,7 @@ describe('wrap', () => {
   test('uses override wrapper', () => {
     const wrapperOverride = 'src/wrappers.auth';
     const response = build.wrap(MOCK_FUNCTION_KEY, MOCK_HANDLER, wrapperOverride);
-    expect(response).toEqual(`${BUILD_DIR}/${MOCK_FUNCTION_KEY}.default`);
+    expect(response).toEqual(`${BUILD_DIR}/${MOCK_FUNCTION_KEY}.handler`);
```

Comment:
To avoid conflict with the reserved keyword `default` I migrated the generated code to use the handler export.